gateway-server/src/test/java/org/apache/knox/gateway/services/token/impl/JDBCTokenStateServiceTest.java

```diff
@@ -49,6 +36,25 @@
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
```

Comment:
We need to make sure your IDE is configured properly in terms of imports (i.e. static imports must come first)

Reply:
I reconfigured it so that static comes first, then java.* then org.*. This order matches the original order of the affected files. But I see exceptions, for example in GatewayServer.java, org.* imports are followed by java.* imports.